elastio-stream-kafka/consumer.py

```diff
@@ -34,39 +36,47 @@
     )
 
 res = subprocess.run(
-    ['elastio', 'rp', 'list', '--output-format', 'json'],
-    stdout=subprocess.PIPE, stderr=subprocess.PIPE
+    ['elastio', 'rp', 'list', '--output-format', 'json', '--type', 'stream'],
+    stdout=subprocess.PIPE, stderr=subprocess.DEVNULL
     )
-topic_already_exists = False
+topic_previously_backed_up = False
 rps = [json.loads(rp) for rp in res.stdout.splitlines()]
 for rp in rps[0]:
     if rp['kind']['kind'] == 'Stream':
         rp_name = rp['asset_snaps'][0]['asset_id'].split(':')[-1]
-        if rp_name == args.topic_name:
-            try:
-                end_timestamp = rp['tags']['end_timestamp']
-                topic_already_exists = True
-                break
-            except KeyError:
+        try:
+            if rp_name == args.stream_name and rp['tags']['topic_name'] == args.topic_name:
+                topic_previously_backed_up = True
                 break
+        except KeyError:
+            break
```

Comment:
Why is it `break`? On any random recovery point that is not tagged with `topic_name` you give up? Looks like it should be `continue`.